touchforms/backend/touchcare.py

```diff
@@ -342,32 +124,26 @@ def from_bundle(inst):
 
             return from_bundle(sess.getSessionInstance(*([self.vars.get(k, '') for k in meta_keys] + \
                                                          [to_hashtable(clean_user_data)])))
-    
-    def _get_fixture(self, user_id, fixture_id):
-        query_url = '%(base)s/%(user)s/%(fixture)s' % { "base": settings.FIXTURE_API_URL, 
-                                                        "user": user_id,
-                                                        "fixture": fixture_id }
-        q = query_factory(self.vars.get('host'), self.vars['domain'], self.auth, format="raw")
-        results = q(query_url)
-        parser = KXmlParser()
-        parser.setInput(to_input_stream(results), "UTF-8")
-        parser.setFeature(KXmlParser.FEATURE_PROCESS_NAMESPACES, True)
-        parser.next()
-        return TreeElementParser(parser, 0, fixture_id).parse()
-
-
-def filter_cases(filter_expr, api_auth, session_data=None, form_context=None):
-    session_data = session_data or {}
-    form_context = form_context or {}
-    modified_xpath = "join(',', instance('casedb')/casedb/case%(filters)s/@case_id)" % \
-        {"filters": filter_expr}
 
-    # whenever we do a filter case operation we need to load all
-    # the cases, so force this unless manually specified
-    if 'preload_cases' not in session_data:
-        session_data['preload_cases'] = True
 
-    ccInstances = CCInstances(session_data, api_auth, form_context)
+def process_form_file(auth, form_data, session_data=None, restore=None, needs_sync=False):
```

Comment:
would be great to have some documentation for `process_form_file` and `process_form_xml`